roles/ceph-facts/tasks/set_radosgw_address.yml

```diff
@@ -73,4 +73,4 @@
   with_items: "{{ groups.get(rgw_group_name, []) }}"
   when:
     - inventory_hostname in groups.get(rgw_group_name, [])
-    - rgw_multisite | bool
+    - hostvars[item]["rgw_multisite"] | bool
```

Comment:
you should do the same change here than above

Reply:
@alimaredia